scripts/build_check.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+pipenv run pycodestyle .
+pipenv run pydocstyle .
+mdl .
```

Comment:
Maybe switch to using `bundle exec mdl` with `Gemfile`? Though this raises the complexity a tad bit (with the extra 1-line-long file).

Or we could remove it entirely and switch to the unmaintained python one.

I personally like the latter.

Reply:
Never mind. I think it's just that my ruby/gem installation is busted. Resolving.

Reply:
Hmm this does seem problematic. I'm getting issues with `gem install ...` as well (I have to use sudo) but then installing bundler requires... `gem install bundler`. Perhaps we should just come back to this later?